platform/email/platform-email-impl/src/test/java/org/codice/ddf/platform/email/impl/SmtpClientImplITCaseTest.java

```diff
@@ -80,7 +84,7 @@ public void testSendWithAttachments()
 
     SimpleSmtpServer server = SimpleSmtpServer.start(port);
 
-    SmtpClientImpl emailService = new SmtpClientImpl();
+    SmtpClientImpl emailService = new SmtpClientImpl(null);
```

Comment:
✏️ Mock `EncryptionService` instead of passing in null. Ideally, the constructor wouldn't allow `null` values anyway.